signNverify.rst

```diff
@@ -342,3 +363,122 @@ specifying ``--group-id`` can also verify the container:
    2   |1       |NONE    |JSON.Generic
    3   |1       |NONE    |FS
    Container verified: my_container.sif
+
+*******************************
+X.509 Key / Certificate Support
```

Comment:
Raw keys in PEM format aren't related to X.509, perhaps something like this would be more accurate:

```suggestion
PEM-Formatted Key / X.509 Certificate Support
```